src/pages/contact/index.tsx

```diff
@@ -1,27 +1,51 @@
 import React from "react";
 import Head from "next/head";
+import classnames from "classnames";
 
 import { ContactUsForm } from "../../components/ContactUsForm";
+import { PageBanner } from "../../components/Banner";
+import { DemoCourseCTA } from "../../components/DemoCourse";
+import { PageHeading } from "../../components/PageHeading";
+
+import styles from "./index.module.scss";
 
 export default function Contact() {
   return (
     <React.Fragment>
       <Head>
         <title>Contact us | PrairieLearn</title>
       </Head>
-      <div className="container my-5">
-        <h1 className="display-3">Contact us</h1>
-        <p className="lead mb-5">
-          Want to learn more about how PrairieLearn can help your course or
-          institution? Need to schedule a demo? Interested in managed hosting?
-          Let us know below and someone from our team will be in touch soon.
-        </p>
-        <div className="row justify-content-center">
+      <PageBanner
+        title="Ready to get started?"
+        subtitle="Request your course space here"
```

Comment:
I'm hesitant to make this the first CTA on the page, and also to have this *and* "Contact us!" as separate CTAs. I'd consider using "Contact us" as the `PageBanner`, removing "request your course space...", and either leaving "Need to schedule..." where it is or making that the subtitle.